src/main/java/io/github/moulberry/notenoughupdates/commands/profile/CataCommand.java

```diff
@@ -33,4 +39,18 @@ public void processCommand(ICommandSender sender, String[] args) {
 		GuiProfileViewer.currentPage = GuiProfileViewer.ProfileViewerPage.DUNGEON;
 		super.processCommand(sender, args);
 	}
+	@Override
+	public List<String> addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
```

Comment:
Doesn't CataCommand already extend ViewProfileCommand, which already implements this sort of tab completion.

Reply:
I think you are right, will remove.